packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -525,30 +527,12 @@ const QueryResultCellRenderer = observer(
     };
 
     const findResultValueFromCoordinates = (
-      resultCoordinate: [number, number],
-    ): string | number | boolean | null | undefined => {
-      const rowIndex = resultCoordinate[0];
-      const colIndex = resultCoordinate[1];
-
-      if (
-        !resultState.executionResult ||
-        !(resultState.executionResult instanceof TDSExecutionResult)
-      ) {
-        return undefined;
-      }
-      if (params.columnApi.getColumnState()[colIndex]?.sort === 'asc') {
-        resultState.executionResult.result.rows.sort((a, b) =>
-          getTDSRowRankByColumnInAsc(a, b, colIndex),
-        );
-      } else if (params.columnApi.getColumnState()[colIndex]?.sort === 'desc') {
-        resultState.executionResult.result.rows.sort((a, b) =>
-          getTDSRowRankByColumnInAsc(b, a, colIndex),
-        );
-      }
-      return resultState.executionResult.result.rows[rowIndex]?.values[
-        colIndex
-      ];
-    };
+      rowIndex: number,
+      colName: string,
+    ): string | number | boolean | null | undefined =>
+      resultState.rowData.filter((data) => data.rowNumber === rowIndex)[0]![
```

Comment:
I think instead of filter we should do find
`resultState.rowData.filter`
Also is this part intended `[0]![colName]` ? what is that suppose to mean